.github/workflows/serverless-integration.yml

```diff
@@ -0,0 +1,39 @@
+name: "Serverless Integration Tests"
+
+on:
+  push:
+    paths:
+      - 'cmd/serverless/**'
+      - 'pkg/serverless/**'
+      - 'test/integration/serverless/**'
```

Comment:
What about a change in a package that the serverless build depends on (e.g. a change on the trace agent or the aggregator)? Can that break tests?

Reply:
Yes, it could, but it would have to be a pretty serious breaking API change to affect our tests.

Reply:
(see other comment for rationale)